simcon_project/conversation_templates/models/template_response.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+import uuid
+
+
+class TemplateResponse (models.Model):
```

Comment:
Idk if these are the 'added changes' you mentioned Smitty... maybe I missed a memo. If this is not what you were talking about, why are all these classes being 'added'? Did we not create them before?

Reply:
Ahh I'm starting to think it was because I accidentally pulled the remote models branch into my local models branch, rather than pulling from main.